### PR TITLE
New release

### PR DESCRIPTION
**Pull request by tvaidotas**

### CODE REVIEW

**Comment by tvaidotas** [comment]
👍

File: app/views/BulkUploadFileView.scala.html [hunk 1]
```diff
@@ -72,10 +72,12 @@
 
     <form action="@upScanResponse.uploadFormTemplate.href" method="POST" enctype="multipart/form-data" id="file-upload-form">
 
-        <div id="file-upload-form-group" class="govuk-form-group @{if(form.hasErrors) "govuk-form-group--error"}">
-           @upScanResponse.uploadFormTemplate.fields.map { row =>
-               <input type="hidden" name="@row._1" value="@row._2">
-           }
+        <div id="file-upload-form-group" class="govuk-form-group @{
```
Comment (ian-ralphs) [inline app/views/BulkUploadFileView.scala.html:75]: I don't like this formatting change

File: app/views/DisclosureNotFoundView.scala.html [hunk 2]
```diff
@@ -33,17 +33,16 @@
     @bullets(
         Seq(
             Html(
-                <span>
-                    {messages("updateCase.disclosureNotFound.li1")}
-                    <a class="govuk-link" href={controllers.routes.DisclosureReferenceNumberController.onLoad().url}>{messages("updateCase.disclosureNotFound.link")}</a>
-                </span>.toString()
+                    <span>
+                    {messages("updateCase.disclosureNotFound.li1")}<a class="govuk-link" href={controllers.routes.DisclosureReferenceNumberController.onLoad().url}>{messages("updateCase.disclosureNotFound.link")}</a>
```
Comment (ian-ralphs) [inline app/views/DisclosureNotFoundView.scala.html:37]: I'm not keen on this change either. It is not very readable.

File: app/views/GuidanceView.scala.html [hunk 3]
```diff
@@ -179,7 +178,8 @@
 
     @p(Html("If you have any questions or have not received the demand note within 10 working days then email customsaccountingrepayments@hmrc.gov.uk."))
 
-    <a class="gem-c-contents-list__link govuk-link " data-track-category="contentsClicked" data-track-action="content_item 1" data-track-label="#who-should-claim" data-track-options="{&quot;dimension29&quot;:&quot;Who should claim&quot;}" href="#0">Contents</a>
+    <a class="gem-c-contents-list__link govuk-link " data-track-category="contentsClicked" data-track-action="content_item 1" data-track-label="#who-should-claim" data-track-options="{&quot;dimension29&quot;:&quot;Who should claim&quot;}" href="#0">
+        Contents</a>
```
Comment (ian-ralphs) [inline app/views/GuidanceView.scala.html:182]: again this seems a strange one

File: app/views/UpdateCaseConfirmationView.scala.html [hunk 4]
```diff
@@ -43,13 +43,18 @@ <h1 class="govuk-panel__title">
...
+        <a class="govuk-link" href="mailto:customsaccountingrepayments@@hmrc.gov.uk">
+            customsaccountingrepayments@@hmrc.gov.uk</a>.
     }
 
     <div id="helpImproveServiceLink">
-        @p{
-            <a href=@{appConfig.surveyUrl} class="govuk-link">@{messages("updateCase.confirmation.helpImproveServiceLink")}</a>
-        }
+    @p {
+        <a href=@{
```
Comment (ian-ralphs) [inline app/views/UpdateCaseConfirmationView.scala.html:52]: breaking up the view elements like this doesn't seem to fit with how I would expect to see html elements

File: app/views/UploadAuthorityView.scala.html [hunk 5]
```diff
@@ -70,24 +70,26 @@
...
-            form = form,
-            name = "file",
-            id = "file",
-            label = messages("uploadAuthority.uploadFile"),
-            isPageHeading = false,
-            attributes = Map(
-                "accept" -> appConfig.upScanAcceptedFileTypes,
-                "data-max-file-size" -> appConfig.upScanMaxFileSize.toString,
-                "data-min-file-size" -> appConfig.upScanMinFileSize.toString
+        <div id="file-upload-form-group" class="govuk-form-group @{
```
Comment (ian-ralphs) [inline app/views/UploadAuthorityView.scala.html:73]: and this one

File: app/views/UploadFileView.scala.html [hunk 6]
```diff
@@ -82,10 +82,12 @@
 
     <form action="@upScanResponse.uploadFormTemplate.href" method="POST" enctype="multipart/form-data" id="file-upload-form">
 
-        <div id="file-upload-form-group" class="govuk-form-group @{if(form.hasErrors) "govuk-form-group--error"}">
-           @upScanResponse.uploadFormTemplate.fields.map { row =>
-               <input type="hidden" name="@row._1" value="@row._2">
-           }
+        <div id="file-upload-form-group" class="govuk-form-group @{
+            if(form.hasErrors) "govuk-form-group--error"
```
Comment (ian-ralphs) [inline app/views/UploadFileView.scala.html:86]: and here

File: app/views/UploadSupportingDocumentationView.scala.html [hunk 7]
```diff
@@ -56,29 +56,31 @@ <h2 class="govuk-heading-s">@messages("uploadSupportingDocumentation.typesOfFile
...
-            form = form,
-            name = "file",
-            id = "file",
-            label = messages("uploadSupportingDocumentation.uploadFile"),
-            isPageHeading = false,
-            attributes = Map(
-                "accept" -> appConfig.upScanAcceptedFileTypes,
-                "data-max-file-size" -> appConfig.upScanMaxFileSize.toString,
-                "data-min-file-size" -> appConfig.upScanMinFileSize.toString
+        <div id="file-upload-form-group" class="govuk-form-group @{
```
Comment (ian-ralphs) [inline app/views/UploadSupportingDocumentationView.scala.html:59]: and here

File: app/views/cya/ImporterConfirmationView.scala.html [hunk 8]
```diff
@@ -60,23 +60,34 @@ <h1 class="govuk-panel__title">
...
         @Html(messages("confirmation.whatYouShouldDoNext.p5"))
-        <a class="govuk-link" href="mailto:customsaccountingrepayments@@hmrc.gov.uk">customsaccountingrepayments@@hmrc.gov.uk</a>.
+        <a class="govuk-link" href="mailto:customsaccountingrepayments@@hmrc.gov.uk">
+            customsaccountingrepayments@@hmrc.gov.uk</a>.
     }
 
-    @p{<a href=@{controllers.routes.IndexController.onPageLoad.url} id="discloseAnotherUnderpayment" class="govuk-link">@{messages("confirmation.whatYouShouldDoNext.link")}</a>}
+    @p {
+        <a href=@{
+            controllers.routes.IndexController.onPageLoad.url
```
Comment (ian-ralphs) [inline app/views/cya/ImporterConfirmationView.scala.html:76]: and here

File: app/views/cya/RepresentativeConfirmationView.scala.html [hunk 9]
```diff
@@ -65,23 +64,34 @@ <h1 class="govuk-panel__title">
...
         @Html(messages("confirmation.whatYouShouldDoNext.p5"))
-        <a class="govuk-link" href="mailto:customsaccountingrepayments@@hmrc.gov.uk">customsaccountingrepayments@@hmrc.gov.uk</a>.
+        <a class="govuk-link" href="mailto:customsaccountingrepayments@@hmrc.gov.uk">
+            customsaccountingrepayments@@hmrc.gov.uk</a>.
     }
 
-    @p{<a href=@{controllers.routes.IndexController.onPageLoad.url} id="discloseAnotherUnderpayment" class="govuk-link">@{messages("confirmation.whatYouShouldDoNext.link")}</a>}
+    @p {
+        <a href=@{
+            controllers.routes.IndexController.onPageLoad.url
```
Comment (ian-ralphs) [inline app/views/cya/RepresentativeConfirmationView.scala.html:80]: and here

File: app/views/cya/RepresentativeConfirmationView.scala.html [hunk 10]
```diff
@@ -65,23 +64,34 @@ <h1 class="govuk-panel__title">
...
+        }</a>
+    }
 
     <div id="helpImproveServiceLink">
-        @p{
-            <a href=@{appConfig.surveyUrl} class="govuk-link">@{messages("confirmation.helpImproveServiceLink")}</a>
-            @Html(messages("confirmation.helpImproveServiceLinkTime"))
-        }
+    @p {
+        <a href=@{
```
Comment (ian-ralphs) [inline app/views/cya/RepresentativeConfirmationView.scala.html:88]: and here

File: app/views/general/ErrorTemplate.scala.html [hunk 11]
```diff
@@ -18,11 +18,15 @@
...
 
 @(pageTitle: String, heading: String, message: String)(implicit request: Request[_], messages: Messages)
-@layout(pageTitle = pageTitle) {
-    <h1 class="govuk-heading-xl">@{Text(heading).asHtml}</h1>
-    <p class="govuk-body">@{Text(message).asHtml}</p>
-}
+    @layout(pageTitle = pageTitle) {
+        <h1 class="govuk-heading-xl">@{
+            Text(heading).asHtml
+        }</h1>
```
Comment (ian-ralphs) [inline app/views/general/ErrorTemplate.scala.html:24]: and here

File: app/views/reasons/TextAmendmentView.scala.html [hunk 12]
```diff
@@ -24,34 +24,37 @@
...
-
-        @formHelper(action = formAction) {
-            @inputText(form, "original", "original", messages("amendmentValue.originalAmount"), false,
-                classes = inputClass)
-            @inputText(form, "amended", "amended", messages("amendmentValue.amendedAmount"), false,
-                classes = inputClass)
-            @button("common.button.continue")
-        }
+    @bodyMessage.map { msg => @p {
+        @msg
```
Comment (ian-ralphs) [inline app/views/reasons/TextAmendmentView.scala.html:45]: and here

File: app/views/reasons/WeightAmendmentView.scala.html [hunk 13]
```diff
@@ -24,34 +24,37 @@
...
-
-        @formHelper(action = formAction) {
-            @inputWeight(form, "original", "original", messages("amendmentValue.originalAmount"), false,
-                classes = inputClass)
-            @inputWeight(form, "amended", "amended", messages("amendmentValue.amendedAmount"), false,
-                classes = inputClass)
-            @button("common.button.continue")
-        }
+    @bodyMessage.map { msg => @p {
+        @msg
```
Comment (ian-ralphs) [inline app/views/reasons/WeightAmendmentView.scala.html:45]: and here

File: app/views/RemoveUploadedFileView.scala.html [hunk 14]
```diff
@@ -27,8 +27,8 @@
 @(form: Form[_], index: Index, filename: String, backlink: Call, action: Call)(implicit request: Request[_], messages: Messages)
 
 @legendContent = {
-    @h1("removeUploadedFile.heading")
-    @p(Html(filename))
+@h1("removeUploadedFile.heading")
```
Comment (ian-ralphs) [inline app/views/RemoveUploadedFileView.scala.html:30]: This doesn't line up

File: app/views/TraderAddressCorrectView.scala.html [hunk 15]
```diff
@@ -25,35 +25,35 @@
 
 @(form: Form[_], traderAddress: ContactAddress, backLink: Call)(implicit request: Request[_], messages: Messages)
 
-    @legendContent = {
-        @h1(messages("traderAddressCorrect.h1"))
+@legendContent = {
+@h1(messages("traderAddressCorrect.h1"))
```
Comment (ian-ralphs) [inline app/views/TraderAddressCorrectView.scala.html:29]: this doesn't look right

**Review by ian-ralphs — CHANGES_REQUESTED** [review]

**Pull request closed without merging**